src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx

```diff
@@ -68,29 +68,30 @@ export const ArtworkSidebarPartnerInfo: FC<ArtworkSidebarPartnerInfoProps> = ({
     showInquiry()
   }
 
-  const locationNames =
-    artwork &&
-    partner &&
-    partner.locations &&
-    partner.locations.length > 0 &&
-    // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-    filterLocations(partner.locations)
-
   return (
     <>
       <Separator my={2} />
       <Flex justifyContent="space-between">
         <PartnerContainer>
           {renderPartnerName()}
-          {locationNames && locationNames.length > 0 && (
-            <Flex mt={1}>
-              <LocationIcon />
-              <Flex flexDirection="column">
-                <Text variant="xs" color="black60" pl={1}>
-                  {locationNames.join(", ")}
+          {partner?.cities && partner.cities.length > 0 && (
+            <>
+              {isCBNEnabled ? (
```

Comment:
I'm confused here, in the parent component we check https://github.com/artsy/force/pull/10193/files#diff-aacf74b81201e9b848f011f880a9891dc317f43d2a1c2d3098ad490d3eda6278R113

Reply:
The check on the parent component is to change this component's **place** on the sidebar.
The check on this component is to limit the number of displayed cities.
I guess this will be cleaner if we can get rid of the feature flag for this PR 🤞